builtin/game/register.lua

```diff
@@ -14,6 +14,8 @@ core.unregister_item_raw = nil
 local register_alias_raw = core.register_alias_raw
 core.register_alias_raw = nil
 
+local override_abm_raw = core.override_abm
```

Comment:
To stay consistent, this should follow the previous naming scheme: `core.override_abm_raw` + `= nil`